### ISSUE TITLE
Duplicate call _verify when using @scope

### ISSUE THREAD

**Issue by phamconganh** [ahopkins/sanic-jwt#171]
in decorator scope, decorated_function : _, instance = await _do_protection(*args, **protect_kwargs) -> _check_authentication -> _verify, and then when user_scopes = instance.auth.extract_scopes(request) -> extract_payload -> _verify

**Comment by ahopkins** [comment]
You are correct. Perhaps there is some optimization to be had here. I would welcome a PR if you would be so kind.
